base/ltexpl.dtx

```diff
@@ -356,9 +356,19 @@
 % \changes{v1.2e}{2020/08/19}
 %         {Add \cs{@expl@cs@\meta{thing}@spec@@N}
 %          for \cs{ShowCommand} (gh/373)}
+% \changes{v1.3h}{2024/04/17}
+%         {Rename \cs{@expl@cs@argument@spec@@N} to
+%          \cs{@expl@cs@parameter@spec@@N} (gh/1014)}
+% \changes{v1.3h}{2024/04/17}
+%         {Update name of \pkg{expl3} function}
+% \changes{v1.3h}{2024/04/17}
+%         {Add a kernel-level copy of \cs{cs_parameter_spec:N}}
 %    \begin{macrocode}
 \cs_gset_eq:NN \@expl@cs@prefix@spec@@N \cs_prefix_spec:N
-\cs_gset_eq:NN \@expl@cs@argument@spec@@N \cs_argument_spec:N
+\cs_if_exist:NTF \cs_parameter_spec:N
+  { \cs_gset_eq:NN \@expl@cs@parameter@spec@@N \cs_parameter_spec:N }
+  { \cs_gset_eq:NN \@expl@cs@parameter@spec@@N \cs_argument_spec:N }
+\cs_gset_eq:NN \__kernel_cs_parameter_spec:N \@expl@cs@parameter@spec@@N
```

Comment:
Does the new `\__kernel_cs_parameter_spec:N` need to be undefined in rollback code?

Reply:
I don't think so - this is strictly kernel-internl

Reply:
I don't insist on adding it, but is `\__kernel_cs_parameter_spec:N` more kernel-internal than `\@expl@cs@parameter@spec@@N`? The latter is undefined in rollback code.

Reply:
We usually undefine in rollback if there is a real chance that the new command was used in the past in documents or code. That really isn't the case for `\@expl@cs@parameter@spec@@N` so I gues it doesn't have to be undefined.  -- Guess it doesn't hurt either way